crates/sui-storage/src/indexes.rs

```diff
@@ -155,8 +155,10 @@ pub struct IndexStoreTables {
     /// **milliseconds** since epoch 1/1/1970). A transaction digest is subjectively time stamped
     /// on a node according to the local machine time, so it varies across nodes.
     /// The timestamping happens when the node sees a txn certificate for the first time.
+    ///
+    /// DEPRECATED. DO NOT USE
     #[default_options_override_fn = "timestamps_table_default_config"]
-    timestamps: DBMap<TransactionDigest, u64>,
+    _timestamps: DBMap<TransactionDigest, u64>,
```

Comment:
I think variable name is used as the column name, so we have to suppress the unused warning with `#[allow(dead_code)]`, instead of renaming the table.

Reply:
Right. We cannot rename db tables.

Reply:
When I tested it locally by storing it from a snapshot, it didn't panic. What is the expected behavior if we rename it to say `_timestamp`?

Reply:
@longbowlu can you try restoring from a snapshot with `main`, run for some time, and in place upgrade to this branch? Without the previous table name, in place upgrade should panic when opening the index rocksdb.